tests/Tests/Models/Customer/CustomerType.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Tests\Models\Customer;
+
+class CustomerType
+{
+    public $name;
```

Comment:
Let's declare `string` here, or directly use constructor property promotion

Reply:
Typed properties are only allowed since PHP 7.4 

Constructor property promotion is only allowed since PHP 8.0

Composer.json
```json
"require": {
        "php": "^7.1 || ^8.0",
        ...,
}
```